### PR TITLE
Add emsdk dependencies to fix codeflow to installer

### PR DESCRIPTION
**Pull request by marcpopMSFT**

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -90,6 +90,16 @@
       <Sha>a117e262c0c49e71a0017e96a0a1124ec990d05f</Sha>
       <SourceBuild RepoName="emsdk" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.net7.Manifest-8.0.100-preview.2" Version="8.0.0-preview.2.23108.1">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>a117e262c0c49e71a0017e96a0a1124ec990d05f</Sha>
+      <SourceBuild RepoName="emsdk" ManagedOnly="true" />
```
Comment (mmitche) [inline eng/Version.Details.xml:96]: Only one of these should be marked as SourceBuild (the one above suffices)

Comment (mmitche) [inline eng/Version.Details.xml:96]:
```suggestion
```

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -90,6 +90,16 @@
...
+    </Dependency>
+    <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-8.0.100-preview.2" Version="8.0.0-preview.2.23108.1">
+      <Uri>https://github.com/dotnet/emsdk</Uri>
+      <Sha>a117e262c0c49e71a0017e96a0a1124ec990d05f</Sha>
+      <SourceBuild RepoName="emsdk" ManagedOnly="true" />
```
Comment (mmitche) [inline eng/Version.Details.xml:101]:
```suggestion
```

**Comment by marcpopMSFT** [comment]
@mmitche please review. I think this is needed to fix codeflow into installer which hasn't gone in many days now.

**Review by mmitche — APPROVED** [review]

**Pull request merged**